upstream/CVE-2021-25741.json

```diff
@@ -0,0 +1,113 @@
+{
+  "id": "CVE-2021-25741",
+  "modified": "2021-09-13T20:58:56Z",
+  "published": "2021-09-13T20:58:56Z",
+  "summary": "Symlink Exchange Can Allow Host Filesystem Access",
+  "details": "A security issue was discovered in Kubernetes where a user may be able to create a container with subpath volume mounts to access files \u0026 directories outside of the volume, including on the host filesystem.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubelet"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.19.0"
+            },
+            {
+              "last_affected": "1.19.14"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubelet"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.20.0"
+            },
+            {
+              "last_affected": "1.20.10"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubelet"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.21.0"
+            },
+            {
+              "last_affected": "1.21.4"
```

Comment:
```suggestion
              "fixed": "1.21.5"
```

```bash
➜  kubernetes git:(master) git tag --contains f5a6d212162ae877b690bca41b7e6c8bab0cef1d | sort -V | head -n 2
v1.21.5
v1.21.6
```

Reply:
fixed in https://github.com/aquasecurity/vuln-list-k8s/pull/87/commits/b912ab678b3386f8a6be8e8739eee2f421f596bc